server/src/main/java/org/elasticsearch/common/settings/Setting.java

```diff
@@ -1372,6 +1372,68 @@ public static ByteSizeValue parseByteSize(String s, ByteSizeValue minValue, Byte
         return value;
     }
 
```

Comment:
At the top of this file, we see this section: (https://github.com/elastic/elasticsearch/blob/b7c7faf3affb869c7d5da6fe3c1aaa9b8d494814/server/src/main/java/org/elasticsearch/common/settings/Setting.java#L77)
```
* <pre>{@code
 * public enum Color {
 *     RED, GREEN, BLUE;
 * }
 * public static final Setting<Color> MY_BOOLEAN =
 *     new Setting<>("my.color.setting", Color.RED.toString(), Color::valueOf, SettingsProperty.NodeScope);
 * }
 * </pre>
```

This example does not make sense anymore (actually it didn't before because the setting name suggests a bool setting, and the snippet wouldn't compile)
However, I was not sure what to replace it with. All suggestions are welcome!